### ISSUE TITLE
Incorrect rendering of landing page of recent ZIMs in jQuery mode

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#552]
At least in `wikipedia_en_history_maxi_2019-08.zim`, the landing page in jQuery mode looks like the top image (and no further images are displayed), whereas it should look something like the bottom image (and have many more images as you scroll down). It works fine in SW mode. So it's a stylesheet injection issue, I think. Tested in Edge 44 and Edge 78 (Chromium).

![image](https://user-images.githubusercontent.com/4304337/63226868-c80dfc80-c1d7-11e9-9e55-b1a501b7b709.png)

![image](https://user-images.githubusercontent.com/4304337/63226844-78c7cc00-c1d7-11e9-8b81-8dbaf457a85d.png)

**Comment by kelson42** [comment]
I confirm the bug is still there.

**Comment by kelson42** [comment]
@Jaifroid Considering that no new dev is foreseen fornthe JQuery mode, would that not be more appropriate to close this ticket?

**Comment by Jaifroid** [comment]
Agreed, and fixing this would require intervening in the HTML of the landing page at reader level, which is against the philosophy of this Repo.
